src/mongo-store.ts

```diff
@@ -0,0 +1,124 @@
+import {IActor, IPermissionStore, IPrivilegeManaged, PermissionsMetaData} from "./types";
+import {Role} from "./am-i-allowed";
+
+import mongoose from "mongoose";
```

Comment:
i don't find mongoose dependency necessary. It doesn't add anything useful you can't do with the official mongo driver. i prefer minimum dependencies. Why did you think it's necessary?

Reply:
I don't find it necessary either. Frankly, this was almost 2 years ago and I have moved on from using my fork of this project. Your project was a great inspiration though, and I am glad you decided to share it with others.

I think the PR should be closed for the time being, for I have no interest in converting the code to the official mongo driver.

In retrospect, I see a lot of issues with the current repository. I'd like to see them fixed first before this PR. Maintenance should always be priority.